slf4j-bridge/src/main/scala/org/slf4j/impl/ZioLoggerFactory.scala

```diff
@@ -0,0 +1,32 @@
+package org.slf4j.impl
+
+import org.slf4j.{ ILoggerFactory, Logger }
+import zio.ZIO
+import zio.logging.Logging
+
+import java.util.concurrent.ConcurrentHashMap
+import scala.jdk.CollectionConverters._
+
+class ZioLoggerFactory extends ILoggerFactory {
+  private var runtime: zio.Runtime[Logging] = null
+  private val loggers                       = new ConcurrentHashMap[String, Logger]().asScala
+
+  def attachRuntime(runtime: zio.Runtime[Logging]): Unit =
+    this.runtime = runtime
+
+  private[impl] def run[A](f: ZIO[Logging, Nothing, A]): Unit =
```

Comment:
Delete the `A` type parameter and accept `ZIO[Logging, Nothing, Any]`, instead. Clearer expression of the same idea.